app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -1181,12 +1264,28 @@ class AdvancedFilter extends React.Component {
                   onChange={event => {
                     this.changeCombinationValue(statementIndex, combinationIndex, { name: combinationValue.name, value: event.target.value });
                   }}>
-                  {this.getCombinationFilter(filter, combinationValue.name).options.map((option, o_index) => {
+                  {combinationFilter.options.map((option, o_index) => {
                     return <option key={o_index}>{option}</option>;
                   })}
                 </Form.Control>
               )}
-              {this.getCombinationFilter(filter, combinationValue.name).type === 'date' && (
+              {combinationFilter?.type === 'multi' && (
```

Comment:
Seeing some weird alignment issues here with the + button.  It should stay inline with the tooltip icon and the minus button

<img width="927" alt="Screen Shot 2021-11-17 at 5 08 50 PM" src="https://user-images.githubusercontent.com/35042815/142291186-66425282-e74a-4ff1-a986-0b30d001790a.png">

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/a3c111454fe6f32302d2c03e60aa2ffe5915f044